tests/CMakeLists.txt

```diff
@@ -108,9 +125,10 @@ if (BROKER_PYTHON_BINDINGS)
   make_python_test(ssl-tests)
   make_python_test(store)
   make_python_test(topic)
-  make_python_test(broker-cluster-benchmark
-                   $<TARGET_FILE:broker-cluster-benchmark>)
-
-  # allow some extra time for the benchmark integration test
-  set_tests_properties(python-broker-cluster-benchmark PROPERTIES TIMEOUT 120)
+  # TODO: re-enable after updating generator files or adding backwards compatiblity
```

Comment:
Ah, actually forgot about this one. Sorry! After skimming through the changes: backwards compatibility definitely isn't straightforward/easy because Broker's message types changed. In particular `internal_message`. So if you're ok with not being backwards compatible, I think plugging in new data is the better option. This integration test was added as regression test for a recording provided by @timwoj (from generated fake DNS data). If one of you could generate a new recording with the ALM branch, that'd be most welcome. 🙂

Reply:
@Neverlord can do any quick sanity checks to see whether message recording is now fundamentally broken ?

I can successful configure a cluster and record data against `master`, but the same setup using this branch generates this recording:

https://www.dropbox.com/s/4oov3bykh7h3yaf/zeek-cluster-recording.zip?dl=0

In either `master` or this branch, the `broker.conf` has an odd `heartbeat-interval = 0d` setting that leads to this error:

```
$ ./bin/broker-cluster-benchmark --mode=generate-config /home/jon/tmp/zeek-cluster-recording/logger /home/jon/tmp/zeek-cluster-recording/manager /home/jon/tmp/zeek-cluster-recording/proxy /home/jon/tmp/zeek-cluster-recording/worker
unable to parse "/home/jon/tmp/zeek-cluster-recording/logger/broker.conf": parser_error(unexpected_character, ({"column" = 26, "line" = 28}))
```

I can manually fudge it to `0s`, but don't recall if there was a better fix/explanation for that and still get further problems in the branch: the `messages.dat` doesn't all expected data.

```
$ ./bin/broker-cluster-benchmark --mode=generate-config /home/jon/tmp/zeek-cluster-recording/logger /home/jon/tmp/zeek-cluster-recording/manager /home/jon/tmp/zeek-cluster-recording/proxy /home/jon/tmp/zeek-cluster-recording/worker > cluster.conf

$ ./bin/broker-cluster-benchmark -c cluster.conf -v
no node expects to receive any data
```

Reply:
Looks like it's broken from the output. I'll look into it.